.gitignore

```diff
@@ -28,3 +28,6 @@ cabal.project.local~
 ### Nix ###
 result*
 .stack-to-nix.cache
+
+### cardano-node ###
+lib/byron/test/data/cardano-node/genesis.json
```

Comment:
Hm, that might be a bit confusing if we'll need to modify genesis.json.

Reply:
yes, working on it

Reply:
reverted in https://github.com/input-output-hk/cardano-wallet/pull/1506/commits/c1b1c8e42e57016eb3e42d3837974ca698fc6142